radio/src/pulses/pulses_arm.h

```diff
@@ -86,7 +74,7 @@ PACK(struct Dsm2TimerPulsesData {
 });
 #endif
 
-#if defined(INTMODULE_USART) || defined(EXTMODULE_USART)
+#if defined(INTMODULE_USART) || defined(EXTMODULE_USART) || defined(PXX2)
```

Comment:
TODO: this file should not be included when building the bootloader, this test shouldn't be needed